articles/aks/cluster-configuration.md

```diff
@@ -281,12 +281,12 @@ To add Mariner to an existing ARM template, you need to add `"osSKU": "mariner"`
 }
 ```
 
-Create this file on your system and fill it with the contents of the Mariner AKS YAML file.
+Create this file on your system and fill it with the contents of the Mariner AKS JSON file.
```

Comment:
Suggest rewording to - Create this file on your system and include the settings defined in the Mariner AKS JSON file.